src/vmm/src/cpu_config/x86_64/static_cpu_templates/t2cl.rs

```diff
@@ -242,7 +242,6 @@ pub fn t2cl() -> CustomCpuTemplate {
             // IA32_ARCH_CAPABILITIES:
             // - Bit 00: RDCL_NO (Intel SDM) / Reserved (AMD APM)
             // - Bit 01: IBRS_ALL (Intel SDM) / Reserved (AMD APM)
-            // - Bit 02: RSBA (Intel SDM) / Reserved (AMD APM)
```

Comment:
Could we keep the line but indicate that the bit is not set? Same below

Reply:
So does it mean you'd prefer to have comments for all the passed-through bits on all the CPUIDs and MSRs? or only for this MSR?

Reply:
At the moment, we have comments only for bits that are set or unset by CPU templates, not for bits that are passed-through.

Reply:
I think it would be useful for somebody reviewing this value to know what each bit is, and why is it enabled/disabled, without having to go through the whole git history. No need to go through all the missing comment, but I think it's worth keeping? Or is the information easily accessible  some other way?

Reply:
> why is it enabled/disabled

These bits are no longer enabled or disabled, instead passed-through.

> No need to go through all the missing comment, but I think those ones are worth keeping? Or is the information easily accessible some other way?

yeah, I can agree with you somewhat. We don't have comments for all the other passed-through bits while we decided to pass through it with more careful investigation than the others.

Reply:
Having these comments in code is sub-optimal solution at the moment. I'd like to add conversion between CPU template format and human-readable format into the helper tool in the future, but we're waiting for feedback from customers right now (whether they really want it).

Reply:
added special comment to lead to this PR.